pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = ["setuptools >= 40.9.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pcw"
+version = "1.0"
```

Comment:
It's not `1.0` but `1.13`.
Also JFYI we're using the version number for tagging containers in [ghcr.io/SUSE/pcw](https://ghcr.io/SUSE/pcw) registry.

Reply:
ahh, I got this from setup.py where was 1.0 :D